app/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from .views import Userview
+
+ 
+urlpatterns = [
+    path('api/', Userview.as_view()),
```

Comment:
route name should be proper. api doesn't make any sense to me....
can be like api/users